utils/gym_utils.py

```diff
@@ -92,13 +95,23 @@ def create_unified_flexiblearmenv_and_controller(create_controller=False):
         n_seg_estimator=n_seg_mpc,
         sim_time=1.3,
         dt=0.01,
-        qa_range_start=np.array([np.pi // 6, np.pi // 6, np.pi // 6]),
-        qa_range_end=np.array([np.pi // 2, np.pi // 2, np.pi // 2]),
+        qa_range_start=np.array([-np.pi/2, 0., -np.pi+0.05]),
+        qa_range_end=np.array([3*np.pi/2, np.pi, np.pi-0.05]),
         contr_input_states=StateType.ESTIMATED,  # "real" if the n_seg is the same for the data and control env
         sim_noise_R=np.diag([*R_Q, *R_DQ, *R_PEE]),
         render_mode="human",
     )
-    env = FlexibleArmEnv(env_options)
+    # Wall obstacle
+    if add_wall_obstacle:
+        # Wall obstacle
+        w = np.array([0., 1., 0.])
+        b = np.array([0., -0.15, 0.5])
+        wall = WallObstacle(w, b)
+    else:
+        wall = None
+
+    # Create environment
+    env = FlexibleArmEnv(env_options, obstacle=wall)
     # -------------------------------------
 
     if create_controller:
```

Comment:
Here we will wait for @RudolfReiter to add the wall obstacle to the NMPC, right?